### PR TITLE
add pgsql and mssql

### PR DESCRIPTION
**Pull request by panawala**
* add mssql input
* add pgsql input

### CODE REVIEW

**Comment by messixukejia** [comment]
1、changelog
2、link issue

**Comment by henryzhx8** [comment]
good

**Comment by panawala** [comment]
close #295 #296

**Comment by henryzhx8** [comment]
Why using nested directory? If so, it seems that plugin mysql should also belong to the rdb directory.

File: test/case/behavior/input_mssql/ilogtail-e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+boot:
+  category: docker-compose
+ilogtail:
+  config:
+    - name: input-mssql-case
+      content:
+        - '{"inputs":[{"detail":{"Address":"mssql","CheckPoint":true,"CheckPointColumn":"id","CheckPointColumnType":"int","CheckPointSavePerPage":true,"CheckPointStart":"0","DataBase":"LogtailTest","IntervalMs":1000,"Limit":true,"MaxSyncSize":100,"PageSize":100,"Password":"MSsqlpa#1word","StateMent":"select * from LogtailTestTable where id > ? ORDER BY id","User":"sa"},"type":"service_mssql"}]}'
```
Comment (henryzhx8) [inline test/case/behavior/input_mssql/ilogtail-e2e.yaml:21]: Change json config to yaml.

File: test/case/behavior/input_pgsql/ilogtail-e2e.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+boot:
+  category: docker-compose
+ilogtail:
+  config:
+    - name: input-pgsql-case
+      content:
+        - '{"inputs":[{"detail":{"Address":"pgsql","CheckPoint":true,"CheckPointColumn":"id","CheckPointColumnType":"int","CheckPointSavePerPage":true,"CheckPointStart":"0","DataBase":"postgres","IntervalMs":1000,"Limit":true,"MaxSyncSize":100,"PageSize":100,"Password":"postgres","StateMent":"select * from specialalarmtest where id > $1","User":"postgres"},"type":"service_pgsql"}]}'
```
Comment (henryzhx8) [inline test/case/behavior/input_pgsql/ilogtail-e2e.yaml:21]: Change json config to yaml.

File: plugins/input/rdb/rdb.go [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+			}
+			if m.MaxSyncSize > 0 && (startOffset+rowCount) >= m.MaxSyncSize {
+				break
+			}
+			if rowCount < m.PageSize {
+				break
+			}
+			select {
+			case <-m.Shutdown:
+				// repush a object and break
```
Comment (henryzhx8) [inline plugins/input/rdb/rdb.go:262]: fix typo

File: plugins/input/rdb/pgsql/pgsql.go [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+	_ "github.com/jackc/pgx/v4/stdlib"
+
+	"github.com/alibaba/ilogtail"
+	"github.com/alibaba/ilogtail/pkg/util"
+	"github.com/alibaba/ilogtail/plugins/input/rdb"
+)
+
+type Pgsql struct {
+	rdb.Rdb
```
Comment (henryzhx8) [inline plugins/input/rdb/pgsql/pgsql.go:32]: Exported Parameters should better be declared explicitly, which you can assign to `rdb.Rdb` in Init.

File: test/case/behavior/input_pgsql/ilogtail-e2e.yaml (hunk 2 above)
Comment (panawala) [reply test/case/behavior/input_pgsql/ilogtail-e2e.yaml]: done

File: plugins/input/rdb/pgsql/pgsql.go (hunk 4 above)
Comment (panawala) [reply plugins/input/rdb/pgsql/pgsql.go:30]: 精简了下Rdb结构的字段，只留了必须的字段

File: CHANGELOG.md [hunk 5]
```diff
@@ -35,3 +35,15 @@ your changes, such as:
 - [public] [both] [updated] add a new feature
 
 ## [Unreleased]
+- [public] [both] [updated] change CGO ProcessLogs API to ProcessLog
```
Comment (messixukejia) [inline CHANGELOG.md:38]: 这些已经随版本发布清掉了，只需提交自己的

Comment (panawala) [reply CHANGELOG.md]: done

File: plugins/input/rdb/pgsql/pgsql.go [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+	return m.Rdb.Collect(collector, nil)
+}
+
+// Stop stops the services and closes any necessary channels and connections
+func (m *Pgsql) Stop() error {
+	return m.Rdb.Stop()
+}
+
+func init() {
+	ilogtail.ServiceInputs["service_pgsql"] = func() ilogtail.ServiceInput {
```
Comment (messixukejia) [inline plugins/input/rdb/pgsql/pgsql.go:110]: 加下开发文档

**Pull request merged**